src/Stripe.net/Services/SubscriptionItems/SubscriptionItemSharedOptions.cs

```diff
@@ -38,5 +39,11 @@ public abstract class SubscriptionItemSharedOptions : BaseOptions
         /// </summary>
         [JsonProperty("quantity")]
         public long? Quantity { get; set; }
+
+        /// <summary>
+        /// Ids of the tax rates to apply to this subscription item.
+        /// </summary>
+        [JsonProperty("tax_rates")]
+        public List<TaxRate> TaxRates { get; set; }
```

Comment:
`tax_rates` here should be List of string

Reply:
huh does `TaxRate` here actually represent a String?

Reply:
no it does not